### PR TITLE
OHRM5X-1395: Fix performance - trackers, tracker logs related issues

### PR DESCRIPTION
**Pull request by devishke-orange**
- OHRM5X-1430: [Performance-Manage Reviews] UI issues related to the list view table.
- OHRM5X-1395: [Performance-KPIs] List view action buttons are not aligned properly.
- OHRM5X-1476: [Performance- Employee Tracker][Mobile] User unable to view or add logs for employees
- OHRM5X-1465: [Performance- Tracker logs] Add performance tracker logs screen UI issues.

Review evaluation commits move to [separate PR](https://github.com/Super-Chama/orangehrm/pull/54)

### CODE REVIEW

File: src/client/src/orangehrmPerformancePlugin/components/FinalEvaluation.vue [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+      editable,
+      formattedCompletedDate,
+    };
+  },
+  data() {
+    return {
+      rules: {
+        completedDate: [required, validDateFormat()],
+        finalRating: [required, digitsOnlyWithDecimalPoint],
+        // TODO add min max rules
```
Comment (RajithaKumara) [inline src/client/src/orangehrmPerformancePlugin/components/FinalEvaluation.vue:124]: TODO

File: src/client/src/orangehrmPerformancePlugin/components/FinalEvaluation.vue [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+      width: 30%;
+      margin-right: 2.4rem;
+    }
+
+    &-rating {
+      width: 10%;
+      margin-right: 2.4rem;
+      //overflow: hidden;
+      //white-space: nowrap;
+      //text-overflow: ellipsis;
```
Comment (RajithaKumara) [inline src/client/src/orangehrmPerformancePlugin/components/FinalEvaluation.vue:159]: *

File: src/client/src/orangehrmPerformancePlugin/components/tracker-log-card.scss [hunk 3]
```diff
@@ -69,6 +69,7 @@
 
     &-text {
       font-weight: 700;
+      font-size: 21px;
```
Comment (RajithaKumara) [inline src/client/src/orangehrmPerformancePlugin/components/tracker-log-card.scss:72]: bug fix?

File: src/client/src/orangehrmPerformancePlugin/pages/ReviewEvaluate.vue [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+      required: true,
+    },
+    isReviewer: {
+      type: Boolean,
+      default: false,
+    },
+  },
+  setup(props) {
+    const http = new APIService(window.appGlobal.baseUrl, '');
+    // TODO workflow
```
Comment (RajithaKumara) [inline src/client/src/orangehrmPerformancePlugin/pages/ReviewEvaluate.vue:130]: TODO

File: src/client/src/orangehrmPerformancePlugin/components/tracker-log-card.scss (hunk 3 above)
Comment (devishke-orange) [reply src/client/src/orangehrmPerformancePlugin/components/tracker-log-card.scss:72]: yes. approved by Kavinda.

**Review by RajithaKumara — APPROVED** [review]

**Pull request merged**